docs/state-of-progressive-decentralization.md

```diff
@@ -56,6 +56,56 @@ This applies only to Arbitrum AnyTrust chains like Arbitrum Nova.
 :::
 
 - **Description**: AnyTrust chains like Arbitrum Nova rely on a permissioned committee to store the chain's data and provide it on demand.
-- **Current status**: 7-member committee. The Arbitrum Nova chain has a 7-party DAC, whose members can be seen [here](https://developer.arbitrum.io/public-chains#arbitrum-chains-summary). Governance has the ability to remove or add members to the committee.
+- **Current status**: 7-member committee. The Arbitrum Nova chain has a 7-party DAC, whose members can be seen [here](#data-availability-committee-members). Governance has the ability to remove or add members to the committee.
 - **Risks**: If 6 of the 7 committee members in conjunction with the Sequencer behave maliciously and collude, the safety of the system can be compromised.
 - **Changes to Current status**: The Arbitrum governance system (see #1) currently has the power to change the DAC, such as by adding or removing members or modifying the power it has over the system.
+
+### Allowlisted validators
+
+These are the current [allowlisted validators](#2-validator-ownership) for every public Arbitrum chain:
+
+#### Arbitrum One
+
+- [0x0fF813f6BD577c3D1cDbE435baC0621BE6aE34B4](https://etherscan.io/address/0x0fF813f6BD577c3D1cDbE435baC0621BE6aE34B4)
```

Comment:
Let's use the AddressExplorerLink component for these (like we do on deployment-addresses.md)